src/main/java/Duke.java

```diff
@@ -28,19 +25,65 @@ public static void main(String[] args) {
                 break;
             } else if (word.equals("list")) {
                 for (int i = 0; i < lists.size(); i++) {
-                    System.out.println(i + 1 + ".[" + getStatus(lists.get(i).isDone) + "] " + lists.get(i).description);
+                    System.out.println(i + 1 + "." + lists.get(i).toString());
                 }
             } else if (word.contains("done")) {
                 String numStr = word.substring(word.indexOf(" ") + 1);
                 int num = Integer.parseInt(numStr);
                 System.out.println("Nice! I've marked this task as done:");
-                Task d = new Task(lists.get(num - 1).description);
-                lists.get(num - 1).isDone = d.markAsDone();
+                lists.get(num - 1).markAsDone();
+                System.out.println(lists.get(num - 1).toString());
+
+
+            } else if (word.contains("deadline")) {
+                String sentence = word.substring(word.indexOf(" ") + 1);
+                String title = sentence.substring(0, sentence.indexOf("/") - 1);
+                String date = word.substring(word.indexOf("by") + 3);
+
+                Task t = new Deadline(title, date);
+                t.toString();
+                lists.add(t);
+
+                System.out.println("Got it. I've added this task:");
+                System.out.println("  " + t);
+                if (lists.size() == 1) {
+                    System.out.println("Now you have " + lists.size() + " task in the list.");
+                } else {
+                    System.out.println("Now you have " + lists.size() + " tasks in the list.");
+                }
+
+            } else if (word.contains("todo")) {
+                String sentence = word.substring(word.indexOf(" ") + 1);
 
-            } else {
-                Task t = new Task(word);
+                Task t = new Todo(sentence);
```

Comment:
Todo, Deadline and Event classes are missing, do remember to upload their .java files!